src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/AbstractMethodOrPropertyOrEventSymbolReferenceFinder.cs

```diff
@@ -18,43 +18,54 @@ protected AbstractMethodOrPropertyOrEventSymbolReferenceFinder()
         {
         }
 
-        protected override async Task<ImmutableArray<ISymbol>> DetermineCascadedSymbolsAsync(
+        protected override async Task<ImmutableArray<(ISymbol symbol, FindReferencesCascadeDirection cascadeDirection)>> DetermineCascadedSymbolsAsync(
             TSymbol symbol,
             Solution solution,
             IImmutableSet<Project>? projects,
             FindReferencesSearchOptions options,
+            FindReferencesCascadeDirection cascadeDirection,
             CancellationToken cancellationToken)
         {
             // Static methods can't cascade.
-            if (!symbol.IsStatic)
+            if (symbol.IsStatic)
+                return ImmutableArray<(ISymbol symbol, FindReferencesCascadeDirection cascadeDirection)>.Empty;
+
+            if (symbol.IsImplementableMember())
+            {
+                // We have an interface method.  Walk down the inheritance hierarchy and find all implementations of
+                // that method and cascade to them.
+                var result = cascadeDirection.HasFlag(FindReferencesCascadeDirection.Down)
+                    ? await SymbolFinder.FindMemberImplementationsArrayAsync(symbol, solution, projects, cancellationToken).ConfigureAwait(false)
+                    : ImmutableArray<ISymbol>.Empty;
+                return result.SelectAsArray(s => (s, FindReferencesCascadeDirection.Down));
```

Comment:
💡 Would be good to have a comment explaining that this intentionally coerces `cascadeDirection` to `FindReferencesCascadeDirection.Down`